lib/blueprinter/field.rb

```diff
@@ -11,4 +11,14 @@ def initialize(method, name, extractor, options = {})
   def extract(object, local_options)
     extractor.extract(method, object, local_options, options)
   end
+
+  def skip?(object, local_options)
```

Comment:
Argument list would be altered to look like: `skip?(object, local_options, blueprint)`

Then you can place your above code here in this method:
```
    %i[if unless].each do |condition|
      if options[condition].is_a?(Symbol)
        options[condition] = blueprint.method(options.fetch(condition))
      end
    end
```